test/testenv.py

```diff
@@ -43,6 +43,12 @@ def get_testid(self):
 
         return name
 
+    def get_relative_filename(self, relative):
```

Comment:
This ends up as an internal method only, so, respecting the naming convention, this should have a leading underscore. I'll leave it up to you on how and if you want to do that here though.

Reply:
I'm not sure if I'll need it later, so I'll leave it as-is for now.